[Space] fix for if copyright key is none

owocado opened the pull request:
also fuck nasa.

owocado committed:
fuck nasa

PredaaA commented on space/core.py:
```diff
@@ -114,6 +114,11 @@ async def apod_text(self, data: dict, context: Union[commands.Context, discord.T
             return "Astronomy Picture of the Day: `Impossible to get Nasa API.`"
 
         details = data["explanation"]
+        try:
+            credits = data["copyright"]
+        except KeyError:
+            credits = None
+            pass
```
```suggestion
```

PredaaA commented on space/core.py:
```diff
@@ -131,7 +136,7 @@ async def apod_text(self, data: dict, context: Union[commands.Context, discord.T
             )
             em.set_image(url=data["url"])
-            em.set_footer(text="Image Credits: {} • Today is {}".format(data["copyright"], data["date"]))
+            em.set_footer(text="Image Credits: {} • Today is {}".format(credits, data["date"]))
```
```suggestion
            em.set_footer(text="Image Credits: {} • Today is {}".format(data.get("copyright", "Unknown"), data["date"]))
```

PredaaA reviewed (changes requested):
A `.get` would be better here. Also "Unknown" instead of "None" may be more user friendly.

owocado committed:
process what Preda suggested

https://discord.com/channels/133049272517001216/133251234164375552/747781167012380773

PredaaA committed:
Use .get instead of try except

owocado commented:
~~Hi @PredaaA ,~~

~~the reason I didn't use `.get()` is because it would make it inconsistent with rest of the format on how data is fetched from API, because at all places the `data[""]` format is used. Thats why. :P~~

Pls ignore this and see below commit kthx. Thank you Preda. <3

owocado committed:
Merge pull request #1 from PredaaA/patch-4

Use .get instead of try except
